ui/src/components/view/InfoCard.vue

```diff
@@ -398,7 +398,12 @@
           <div class="resource-detail-item__label">{{ $t('label.templatename') }}</div>
```

Comment:
@Pearl1594 should we also have label read ISO with something like
```suggestion
          <div class="resource-detail-item__label">{{ resource.isoid ? $t('label.iso') : $t('label.templatename') }}</div>
```

Reply:
Thanks @shwstppr. Makes sense.. I've updated the PR.